src/Slick/Pandoc.hs

```diff
@@ -1,47 +1,64 @@
+{-# LANGUAGE DeriveAnyClass    #-}
+{-# LANGUAGE DeriveGeneric     #-}
 {-# LANGUAGE OverloadedStrings #-}
 
 module Slick.Pandoc
   ( markdownToHTML
   , markdownToHTML'
   , makePandocReader
   , makePandocReader'
-  , loadUsing
-  , loadUsing'
-  , convert
   , html5Options
   , markdownOptions
   , PandocReader
   , PandocWriter
+  , loadUsing
+  , loadUsing'
+  , loadEntity
   ) where
 
-import Control.Lens
-import Control.Monad
-import Data.Aeson
-import Data.Aeson.Lens
-import qualified Data.Text as T
-import Development.Shake hiding (Resource)
-import Text.Pandoc
-import Text.Pandoc.Highlighting
-import Text.Pandoc.Shared
+import           Control.Lens
+import           Control.Monad
+import           Data.Aeson
+import           Data.Aeson.Lens
+import qualified Data.Text                  as T
+import           Development.Shake
+import           Development.Shake          hiding (Resource)
+import           Development.Shake.Classes
+import           Development.Shake.FilePath
+import           GHC.Generics               hiding (Meta)
+import           Text.Pandoc
+import           Text.Pandoc.Highlighting
+import           Text.Pandoc.Shared
+
+import           Slick.Build
+
+--------------------------------------------------------------------------------
+
+type PandocReader textType = textType -> PandocIO Pandoc
+type PandocWriter = Pandoc -> PandocIO T.Text
 
 -- | Reasonable options for reading a markdown file
 markdownOptions :: ReaderOptions
-markdownOptions = def { readerExtensions = exts }
- where
-  exts = mconcat
-    [ extensionsFromList
-      [ Ext_yaml_metadata_block
-      , Ext_fenced_code_attributes
-      , Ext_auto_identifiers
-      ]
-    , githubMarkdownExtensions
-    ]
+markdownOptions =
```

Comment:
I believe this should be configurable by default. (And default `markdownOptions` should be kicked to `example/`.)

Reply:
so, better to remove completely?